src/api/index.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+
+import { API_KEY, BASE_API_URL } from '../constants';
+
+const UNITS = 'metric';
+const FORECAST_QUERY_URL = `${BASE_API_URL}/forecast`;
+
+export const get5DaysForecastByCoords = async (location) => {
+  const { status, statusText, data } = await axios.get(
+    FORECAST_QUERY_URL,
+    {
+      params: {
+        APPID: API_KEY,
+        units: UNITS,
+        lat: location.latitude,
+        lon: location.longitude,
+      }
+    }
+  );
+
+  return { status, statusText, data };
```

Comment:
If this wasn't transformed at all then why you deconstruct this and then construct new object?

Reply:
I just wanted to return some of the fields from the response. I will look for a better way of doing this.

Reply:
ok